stacks/utils/getFunctionProps.ts

```diff
@@ -1,16 +1,14 @@
 import { use } from "sst/constructs";
 
-import { ConfigStack } from "./ConfigStack";
-import { EventBusStack } from "./EventBusStack";
-import { SchedulerStack } from "./SchedulerStack";
-import { StorageStack } from "./StorageStack";
+import { ConfigStack } from "../ConfigStack";
+import { EventBusStack } from "../EventBusStack";
+import { SchedulerStack } from "../SchedulerStack";
+import { StorageStack } from "../StorageStack";
 
-export const getFunctionProps = () => {
+export const getBaseFunctionProps = () => {
```

Comment:
Overall im not sure about that functionProps util at all, not sure that is truly the base one, the migrationDb does not need the eventbridge stuff(bind,permission) and also the scheduler lambda arns is not needed for the migration i guess.

Reply:
I created 4 groups, the base one is just the secrets and runtime, there's one for the db (used by migration fn), one for eventbus (used by scheduleHandler) and one which includes everything